samples/some-assertions.golo

```diff
@@ -0,0 +1,54 @@
+module some.assertions
```

Comment:
Lacks a license header, plus you should use a module name like `golo.samples.Assertions` or similar.